lib/matobjplist.gi

```diff
@@ -1286,3 +1286,330 @@ InstallMethod( NewCompanionMatrix,
     return ll;
   end );
 
+############################################################################
+# Elementary matrix operations
+############################################################################
+
+############################################################################
+##
+#M  MultMatrixRow( <M>, <row>, <scalar> )
+##
+InstallMethod( MultMatrixRow, "for a mutable IsPlistMatrixRep, one row number, and an scalar",
```

Comment:
`IsPlistMatrixRep` is just one implementation, the method is installed for `IsRowListMatrix`.